### PR TITLE
generated scss

### PR DESCRIPTION
**Pull request by yariza**
I generated this using http://sebastianpontow.de/css2compass/
This is pretty rough and definitely needs review; in particular
let me know if the advice on the commented lines should be taken,
and if the @extend variable stuff should be taken out.

### CODE REVIEW

**Comment by yariza** [comment]
ok @Adjective-Object you can check it now

File: client/css/default.scss [hunk 1]
```diff
@@ -1,81 +1,138 @@
...
+$nobel_approx: #b3b3b3;
+$patina_approx: #5e8d87;
+$persian_green_approx: #05b491;
+$shark_approx: #2a2a2a;
+$sunset_orange_approx: #ff5138;
+$white: #fff;
+$white_10: rgba(255,255,255, 0.1);
+$white_50: rgba(255,255,255,0.5);
+$white_8: rgba(255,255,255, 0.08);
+$wild_sand_approx: whitesmoke;
```
Comment (Adjective-Object) [inline client/css/default.scss]: Probably worth calling this `$whitesmoke`, or using `#f5f5f5` for this

File: client/css/default.scss [hunk 2]
```diff
@@ -1,81 +1,138 @@
...
+h2 {
+  font-weight: normal;
+}
+h3 {
+  font-weight: normal;
+}
+h4 {
+  font-weight: normal;
+}
+h5 {
```
Comment (Adjective-Object) [inline client/css/default.scss]: breaking these into separate selectors is uggo

File: client/css/markdown-document.scss [hunk 3]
```diff
@@ -140,771 +478,827 @@ dd {
...
   list-style-type: none;
+  input {
+    float: left;
+    margin: .3em 0 .25em -1.6em;
+    vertical-align: middle;
+  }
+  + .task-list-item {
+    margin-top: 3px;
+  }
 }
```
Comment (Adjective-Object) [inline client/css/markdown-document.scss]: I think this whole highlight section is just a prism stylesheet, we should be able to break this out.

**Pull request merged**